dtest.py

```diff
@@ -249,6 +249,12 @@ def set_dtest_setup_on_function(self, fixture_dtest_setup):
         self.dtest_config = fixture_dtest_setup.dtest_config
         return None
 
+    @pytest.fixture(autouse=True)
+    def cleanup_connections(self):
+        yield None
```

Comment:
Can you add a comment about why this yield is necessary?

Reply:
this is documented when you look up `fixture`, its a core way that this works... w/e is before `yield` is setup, w/e is after `yield` is cleanup.

Reply:
Ah right on - no extra doc necessary then

Reply:
https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended